### PR TITLE
Update Set up your development doc for Medusa CLI section

### LINKED ISSUE #2852: Docs: add a code block to test out Medusa CLI after installing it
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

In the [Medusa CLI section of Set up your development environment](https://docs.medusajs.com/tutorial/set-up-your-development-environment#medusa-cli) documentation, we should add a command to test out that the CLI tool has been installed.

### How can this issue be resolved?

1. Add in the existing block the following line: `medusa -v`

### Are you interested in working on this issue?

- [ ] I would like to fix this issue

### PR DESCRIPTION
**Pull request by debashiscodes**
closes #2852

### CODE REVIEW

File: docs/content/tutorial/0-set-up-your-development-environment.mdx [hunk 1]
```diff
@@ -127,6 +127,12 @@ You can install Medusa’s CLI with the following command:
 npm install @medusajs/medusa-cli -g
 ```
 
+You can check which version of Medusa you have by running the following command:
```
Comment (shahednasser) [inline docs/content/tutorial/0-set-up-your-development-environment.mdx:130]:
```suggestion
You can check that Medusa was installed by running the following command:
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thanks for picking up this issue @DebashisCodes ! Just a small suggested change below.

**Comment by DebashisCodes** [comment]
Thanks, @shahednasser for your review.

**Review by shahednasser — APPROVED** [review]
LGTM! Thanks for your contribution @DebashisCodes

**Pull request merged**